test/test-applications/integrations/TestApplication.CustomSdk/Program.cs

```diff
@@ -74,7 +79,8 @@ public static async Task Main(string[] args)
 
                 using var client = new HttpClient();
                 client.Timeout = TimeSpan.FromSeconds(5);
-                await client.GetStringAsync("https://httpbin.org/get", cancellation.Token);
+                var port = int.Parse(args[3]);
```

Comment:
```suggestion
                var port = int.Parse(args[3], CultureInfo.InvariantCulture);
```

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/4107/commits/e52867251269ffa2c7048f070d8e9ae01352c2e8